tester/src/test/java/com/silanis/esl/sdk/examples/SignableOptionalSignaturesExampleTest.java

```diff
@@ -0,0 +1,24 @@
+package com.silanis.esl.sdk.examples;
+
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class SignableOptionalSignaturesExampleTest {
+
+    @Test
+    public void verifyResult() {
+        SignableOptionalSignaturesExample example = new SignableOptionalSignaturesExample();
+        example.run();
+
+        assertThat("signer1's signable signatures are not set correctly. ", example.signer1SignableSignatures.size(), is(2));
+        assertThat("signer1's signable signature1 is not set correctly. ", example.signer1SignableSignatures.get(0).getSignerEmail(), is(example.email1));
+        assertThat("signer1's signable signature2 is not set correctly. ", example.signer1SignableSignatures.get(1).getSignerEmail(), is(example.email1));
+
+        assertThat("signer2's signable signatures are not set correctly. ", example.signer2SignableSignatures.size(), is(2));
```

Comment:
Use "hasSize" matcher instead of "is"

Reply:
It's directly copied from SignableSignaturesExampleTest - should we keep it consistent or do we want to modify the old example test as well?

Reply:
Let me fix the old one.